File: C#/Example Looping/Uncounted Loop/While.cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            char vartry = 'y';
            int i = 0;

            while (vartry == 'y')
            {
                Console.WriteLine();
                Console.WriteLine("Enter y for Example");
                Console.WriteLine();
                Console.Write("Enter : ");
                vartry = char.Parse(Console.ReadLine());
                i++;
            }
            Console.WriteLine();
            Console.WriteLine("Done");

        }
    }
}

File: Kotlin/Example Looping/Uncounted Loop/While.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var input = Scanner(System.`in`)
    var vartry = true
    var i = 0
    
    while (vartry) {
        println(" ")
        println("Enter y for Example")
        println(" ")
        print("Enter : ")
        var varchc = input.next()

        if( varchc.equals("y") ){
            vartry = true
        }
        else{
            vartry = false
        }
        i++
    }
    println(" ")
    println("Done")

}
